crates/subspace-farmer/src/legacy_multi_plots_farm.rs

```diff
@@ -73,15 +75,15 @@ impl LegacyMultiPlotsFarm {
             enable_dsn_sync,
             enable_farming,
         }: Options<C>,
-        total_plot_size: u64,
+        allocated_space: u64,
         max_plot_size: u64,
         new_plot: impl Fn(usize, PublicKey, u64) -> Result<Plot, PlotError>
             + Clone
             + Send
             + Sync
             + 'static,
     ) -> anyhow::Result<Self> {
-        let plot_sizes = get_plot_sizes(total_plot_size, max_plot_size);
+        let plot_sizes = get_plot_sizes(allocated_space, max_plot_size);
```

Comment:
Hmm, `single_plot_farm` and `plot` wasn't changed in terms of handling plot size. I bet it might be a bug

Reply:
Yes, fixed